sys/cbor/cbor.c

```diff
@@ -92,6 +96,18 @@
 #endif
 
 #ifndef CBOR_NO_FLOAT
+
+/* pack to force aligned access on ARMv7 (buggy GCC) */
+#pragma GCC diagnostic error "-Wcast-align"
```

Comment:
Not sure, I understand it myself - I'm only a caretaker of this PR. But I will try my best.